recipe/build.sh

```diff
@@ -16,7 +16,5 @@ fi
 cmake ${RECIPE_DIR}/../ $extra_args \
                         -DCONDA_BUILD=ON \
                         -DCMAKE_BUILD_TYPE=RelWithDebInfo \
-                        -DLIBRARY_LIB=$CONDA_PREFIX/lib \
-                        -DLIBRARY_INC=$CONDA_PREFIX/include \
```

Comment:
We need these in. If we build locally (not though conda) it allows us to link the system libraries. Without them it only knows to look in the one place.

Reply:
Nope.

- you should not use `recipe/build.sh` to "build locally (not though conda)"
- `LIBRARY_LIB` and `LIBRARY_INC` are unused variables
  + see the CMake Warnings about them
  + try to find them anywhere in the codebase

> allows us to link the system libraries. Without them it only knows to look in the one place.

[![citation needed](https://imgs.xkcd.com/comics/wikipedian_protester.png)](https://xkcd.com/285/)

Reply:
Ah you're right. It's `IPP_LIBRARY` and `IPP_INCLUDE` I pass to cmake!